src/core/client/framework/lib/messages.tsx

```diff
@@ -84,6 +84,12 @@ export const INVALID_URL = () => (
   </Localized>
 );
 
+export const INVALID_URL_START = () => (
+  <Localized id="framework-validation-invalidURLStart">
+    <span>Be sure to include http:// or https:// in your URL</span>
+  </Localized>
+);
+
```

Comment:
Maybe call this `INVALID_URL_NOT_HTTP` because in essence we check if the http(s) protocol is used or no.